### PR TITLE
CLOUDP-128898: Provide commands to enable/disable alert settings

### PR DESCRIPTION
**Pull request by fmenezes**
## Proposed changes
Provide commands to enable/disable alert settings

_Jira ticket:_ CLOUDP-128898

Closes #[issue number]

## Checklist

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [ ] I have run `make fmt` and formatted my code

### CODE REVIEW

File: internal/cli/alerts/alerts.go [hunk 1]
```diff
@@ -35,6 +35,8 @@ func Builder() *cobra.Command {
 		ListBuilder(),
 		AcknowledgeBuilder(),
 		UnacknowledgeBuilder(),
+		EnableBuilder(),
+		DisableBuilder(),
```
Comment (gssbzn) [inline internal/cli/alerts/alerts.go:39]: these should be added to https://github.com/mongodb/mongodb-atlas-cli/blob/c45e940f6ecc34b687b1df479c2bb8115514f471/internal/cli/alerts/settings/settings.go#L164-L170 if I'm getting the request correctly 
```suggestion
```

File: internal/store/alert_configuration.go [hunk 2]
```diff
@@ -112,3 +116,17 @@ func (s *Store) UpdateAlertConfiguration(alertConfig *atlas.AlertConfiguration)
 		return nil, fmt.Errorf("%w: %s", errUnsupportedService, s.service)
 	}
 }
+
+// EnableAlertConfiguration encapsulate the logic to manage different cloud providers.
+func (s *Store) EnableAlertConfiguration(projectID, id string, enable bool) (*atlas.AlertConfiguration, error) {
```
Comment (gssbzn) [inline internal/store/alert_configuration.go:121]: instead of passing a boolean could we have dedicated enable/disable dedicated methods?

Comment (fmenezes) [reply internal/store/alert_configuration.go:121]: sure, but why?

Comment (gssbzn) [reply internal/store/alert_configuration.go:121]: it's a bit confusing a method called Enabled which takes false to actually disable, not blocking in any way it's just making it a bit more consumable

File: internal/cli/alerts/settings/disable.go [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+	}
+
+	return opts.Print(r)
+}
+
+// mongocli atlas alerts disable <ID> --projectId projectId.
+func DisableBuilder() *cobra.Command {
+	opts := new(DisableOpts)
+	cmd := &cobra.Command{
+		Use:   "disable <alertId>",
```
Comment (gssbzn) [inline internal/cli/alerts/settings/disable.go:59]:
```suggestion
		Use:   "disable <alertConfigId>",
```

File: internal/cli/alerts/settings/disable.go [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+		Args:  require.ExactArgs(1),
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			return opts.PreRunE(
+				opts.ValidateProjectID,
+				opts.initStore(cmd.Context()),
+				opts.InitOutput(cmd.OutOrStdout(), disableTemplate),
+			)
+		},
+		Annotations: map[string]string{
+			"alertIdDesc": "ID of the alert you want to enable.",
```
Comment (gssbzn) [inline internal/cli/alerts/settings/disable.go:70]:
```suggestion
			"alertConfigIdDesc": "ID of the alert configuration you want to enable.",
```

File: internal/cli/alerts/settings/disable.go [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+
+func (opts *DisableOpts) initStore(ctx context.Context) func() error {
+	return func() error {
+		var err error
+		opts.store, err = store.New(store.AuthenticatedPreset(config.Default()), store.WithContext(ctx))
+		return err
+	}
+}
+
+var disableTemplate = "Alert '{{.ID}}' disabled\n"
```
Comment (gssbzn) [inline internal/cli/alerts/settings/disable.go:44]:
```suggestion
var disableTemplate = "Alert setting '{{.ID}}' disabled\n"
```

File: internal/cli/alerts/settings/disable.go [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+
+	return opts.Print(r)
+}
+
+// mongocli atlas alerts disable <ID> --projectId projectId.
+func DisableBuilder() *cobra.Command {
+	opts := new(DisableOpts)
+	cmd := &cobra.Command{
+		Use:   "disable <alertId>",
+		Short: "Disables one alert for the specified project.",
```
Comment (gssbzn) [inline internal/cli/alerts/settings/disable.go:60]:
```suggestion
		Short: "Disables one alert configuration for the specified project.",
```

File: internal/cli/alerts/settings/enable.go [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+	}
+
+	return opts.Print(r)
+}
+
+// mongocli atlas alerts enable <ID> --projectId projectId.
+func EnableBuilder() *cobra.Command {
+	opts := new(EnableOpts)
+	cmd := &cobra.Command{
+		Use:   "enable <alertId>",
```
Comment (gssbzn) [inline internal/cli/alerts/settings/enable.go:59]:
```suggestion
		Use:   "enable <alertConfigId>",
```

File: internal/cli/alerts/settings/enable.go [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+		Short: "Enables one alert for the specified project.",
```
Comment (gssbzn) [inline internal/cli/alerts/settings/enable.go:60]:
```suggestion
		Short: "Enables one alert configuration for the specified project.",
```

File: internal/cli/alerts/settings/enable.go [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+		Args:  require.ExactArgs(1),
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			return opts.PreRunE(
+				opts.ValidateProjectID,
+				opts.initStore(cmd.Context()),
+				opts.InitOutput(cmd.OutOrStdout(), enableTemplate),
+			)
+		},
+		Annotations: map[string]string{
+			"alertIdDesc": "ID of the alert you want to enable.",
```
Comment (gssbzn) [inline internal/cli/alerts/settings/enable.go:70]:
```suggestion
			"alertConfigIdDesc": "ID of the alert configuration you want to enable.",
```

File: internal/cli/alerts/settings/enable.go [hunk 10]
```diff
@@ -0,0 +1,82 @@
...
+
+func (opts *EnableOpts) initStore(ctx context.Context) func() error {
+	return func() error {
+		var err error
+		opts.store, err = store.New(store.AuthenticatedPreset(config.Default()), store.WithContext(ctx))
+		return err
+	}
+}
+
+var enableTemplate = "Alert '{{.ID}}' enabled\n"
```
Comment (gssbzn) [inline internal/cli/alerts/settings/enable.go:44]:
```suggestion
var enableTemplate = "Alert configuration '{{.ID}}' enabled\n"
```

**Review by gssbzn — COMMENTED** [review]
Some copy changes then LGTM

**Review by gssbzn — APPROVED** [review]
LGTM thanks for doing this

**Pull request merged**